src/Http/Controllers/GithubController.php

```diff
@@ -11,41 +12,47 @@
 
 class GithubController extends Controller
 {
-    public function __construct()
+    private SocialiteManager $socialiteManager;
+
+    /**
+     * @param SocialiteManager $socialiteManager
+     */
+    public function __construct(SocialiteManager $socialiteManager)
     {
+        $this->socialiteManager = $socialiteManager;
```

Comment:
@DarkWasabi - actually since we are moving to php 8, we can use constructor promotion, which is about a billion times than the previous way. https://stitcher.io/blog/constructor-promotion-in-php-8

e.g.

``php
public function __construct(private SocialiteManager $socialiteManager)
{
  // And then no need for any more declarations or assignments
}
```